config/samples/azure_v1alpha2_mysqlaaduser.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: azure.microsoft.com/v1alpha2
+kind: MySQLAADUser
+metadata:
+  name: mysqlaaduser-sample
+spec:
+  server: mysqlserver-sample
+  dbName: mysqldatabase-sample
```

Comment:
There is no `dbname` field at this location in the new CRD?

Reply:
D'oh!